lib/rspecq/reporter.rb

```diff
@@ -116,16 +116,21 @@ def humanize_duration(seconds)
     def flaky_jobs_to_sentry(jobs, build_duration)
       return if jobs.empty?
 
-      Raven.capture_message("Flaky jobs detected", level: "warning", extra: {
-        build: @build_id,
-        build_timeout: @timeout,
-        queue: @queue.inspect,
-        object: self.inspect,
-        pid: Process.pid,
-        flaky_jobs: jobs,
-        flaky_jobs_count: jobs.count,
-        build_duration: build_duration
-      })
+      jobs.each do |job|
+        filename = job.sub(/\[.+\]/, '')
+        Raven.capture_message("Flaky test file: #{filename}", level: "warning", extra: {
+          build: @build_id,
+          build_timeout: @timeout,
+          queue: @queue.inspect,
+          object: self.inspect,
+          pid: Process.pid,
+          job_path: job,
+          build_duration: build_duration,
+        }, tags: {
+            'flaky': true,
```

Comment:
`flaky: true`

Reply:
Just fyi this is ruby is still a symbol not a string.
A string key would have to be declared as such: `'flaky' => true`